webapp/src/css/inbox.less

```diff
@@ -115,226 +141,240 @@ body {
-        color: #262626;
-        margin-top: 0;
-        white-space: normal;
-        font-size: 1rem;
+        padding-top: 0;
+        padding-bottom: 0;
+        color: @label-color;
+        &:hover {
+          background-color: transparent;
+          cursor: default;
+        }
       }
-      .disabled a {
-        color: #999999;
+      .success {
+        color: @ok-color;
       }
-      .divider {
-        margin: 0;
-      }
-      .sync-status {
-        margin-top: 10px;
-        margin-bottom: 10px;
-        a {
-          padding-top: 0;
-          padding-bottom: 0;
-          color: @label-color;
-          &:hover {
-            background-color: transparent;
-            cursor: default;
-          }
-        }
-        .success {
-          color: @ok-color;
-        }
-        .required {
-          color: @error-color;
-        }
+      .required {
+        color: @error-color;
       }
     }
+  }
 
-    .tabs {
-      overflow: hidden;
-      white-space: nowrap;
+  .tabs {
+    overflow: hidden;
+    white-space: nowrap;
 
-      .button-label {
-        top: -5px;
-        position: relative;
-      }
+    .button-label {
+      top: -5px;
+      position: relative;
+    }
 
-      a {
-        border-radius: @radius-size @radius-size 0 0;
-        cursor: pointer;
-        min-width: 80px;
-        padding-left: 5px;
-        padding-right: 5px;
+    a {
+      border-radius: @radius-size @radius-size 0 0;
+      cursor: pointer;
+      min-width: 80px;
+      padding-left: 5px;
+      padding-right: 5px;
 
-        .mm-icon {
-          display: block;
-          padding: 5px 0;
-
-          svg, span:not(.mm-badge-overlay):not(.mm-badge), img {
-            &, &:before {
-              display: block;
-              margin: auto;
-              height: 24px;
-              max-width: 24px;
-              overflow: hidden;
-            }
+      .mm-icon {
+        display: block;
+        padding: 5px 0;
+
+        svg, span:not(.mm-badge-overlay):not(.mm-badge), img {
+          &, &:before {
+            display: block;
+            margin: auto;
+            height: 24px;
+            max-width: 24px;
+            overflow: hidden;
           }
         }
+      }
 
-        &.selected {
-          .button-label,
-          .mm-icon:hover,
-          .resource-icon {
-            .button-text;
-          }
+      &.selected {
+        .button-label,
+        .mm-icon:hover,
+        .resource-icon {
+          .new-nav-button-text;
         }
+      }
 
-        @media (hover: hover) { // Prevents sticky hover state in mobile devices.
-          &:hover:not(.selected) {
-            &.messages-tab {
-              &,
-              .mm-icon-inverse,
-              .mm-icon-inverse:hover,
-              .resource-icon svg * {
-                color: @messages-color;
-                fill: @messages-color;
-              }
+      @media (hover: hover) { // Prevents sticky hover state in mobile devices.
```

Comment:
Test tab action (but not clicking) in mobile phone. Just to make sure the tab doesn't have the `selected` style

Reply:
I've reviewed the behavior from the linked PRs -> https://github.com/medic/cht-core/pull/7192, https://github.com/medic/cht-core/issues/7203 

I've checked both old nav and new nav. But please double check